msteams-platform/toolkit/debug-your-agents-playground.md

```diff
@@ -130,7 +130,7 @@ Agents Playground offers a faster debug experience for applications when compare
    1. Run the following command to install Agents Playground CLI from `npm`:
 
       ```cmd
-      npm install -g @microsoft/microsoft-365-agents-playground
+      npm install -g @microsoft/teams-app-test-tool
```

Comment:
`teams-app-test-tool` is the _old_ name.

the new package is https://www.npmjs.com/package/@microsoft/m365agentsplayground 

/C @qinzhouxu